mlflow/models/evaluation/base.py

```diff
@@ -655,6 +666,7 @@ def evaluate(
     feature_names: list = None,
     evaluators=None,
     evaluator_config=None,
+    custom_metrics=None,
```

Comment:
rm ","

Reply:
This "," is automatically inserted by black. I'm curious why we need to remove it.

Reply:
Is it for preventing this issue?
https://stackoverflow.com/questions/33350454/syntaxerror-with-passing-kwargs-and-trailing-comma

Reply:
I recall there was some issue with it long time ago but don't remember exactly why. If black adds it, and it runs fine, I'm cool with it.